Sources/Apollo/GraphQLOperation.swift

```diff
@@ -1,6 +1,7 @@
 public protocol GraphQLOperation: class {
   static var operationString: String { get }
   static var requestString: String { get }
+  static var operationIdentifier: String { get }
```

Comment:
I'll go ahead and update `apollo-codegen` to produce `String?`s, in keeping with our treatment of `variables`.